src/libgeds/TcpTransport.cpp

```
@@ -179,6 +179,7 @@ bool TcpPeer::processEndpointSend(std::shared_ptr<TcpEndpoint> tep) {
       ctx->hdr.datalen = work->len;
       ctx->hdr.offset = work->off;
       ctx->hdr.type = work->type;
+      ctx->hdr.error = work->error;
```

oh yes we may want to report a delayed error